src/main/java/frc/robot/RobotContainer.java

```diff
@@ -70,18 +75,19 @@ public RobotContainer() {
     DriveConstants.update();
 
     // Create Drivetrain, because every robot will have a drivetrain
-    m_drive = new Drivetrain(m_drivetrainTab, m_swerveModulesTab);
-    m_driver = new GameControllerDriverConfig(m_drive, m_controllerTab, false);
+    m_drive = null;//new Drivetrain(m_drivetrainTab, m_swerveModulesTab);
+    m_driver = null; //new GameControllerDriverConfig(m_drive, m_controllerTab, false);
```

Comment:
don't set the drivetrain to null. I know you need to do this to test on the class bots, but revert this on your PR